### PR TITLE
[Doc] More for results & conclusions

### PR DESCRIPTION
**Pull request by alb3rtobr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1529,23 +1532,24 @@ $ helm del --purge tfm-almacar
...
-* Easy to divide microservices among several teams or developers, as it was our case.
-* Persistence was more difficult than we thought.
-* We increased our knowledge about several cloud technologies than when we started the development.
+Persistence was more difficult than we thought. TODO: complete
 
-Next steps that could be done to improve this application would include:
+Finally, we want to highlight which could be the next steps in this project, if it were possible to continue with the development:
 
-* Improve data model, providing more attributes or relatioships that could be useful taking into account the possible users.
+* Improve data model, providing more attributes or relationships that could be useful taking into account the possible users.
```
Comment (kikogolk) [inline README.md:1556]: I miss tracing and ISTIO

File: README.md [hunk 2]
```diff
@@ -1529,23 +1532,24 @@ $ helm del --purge tfm-almacar
...
+
+We have also experiment how a microservice project can grow easily, being difficult to run it locally, which creates the opportunity for the PaaS vendors which are offering "Kubernetes as a service". Minikube is enough for small projects, but we had resources problems when trying to adapt Elastic, although we have just one replica per deployment.
+
+Using microservices, it has been easier to divide tasks among us. This is an advantage of this kind of architectures, as different teams can take ownership of different services and work in parallel, speeding the application development.
```
Comment (kikogolk) [inline README.md:1550]: I would say in this part that it was great for us becuuse we could practice with technologies we are using right now in the project we are starting.

File: README.md [hunk 3]
```diff
@@ -1529,23 +1532,24 @@ $ helm del --purge tfm-almacar
 
 # 5. Results
 
-TODO
+Summarizing what has been exposed in previous chapters of this document, we should say we have implemented three versions of our application, each one improving its predecessor. As a result after these three iteractions, we have a microservice-based Spring application, implemented to be deployed on a Kubernetes cluster with Helm charts, that could be used as backed for a stock management system.
```
Comment (carlosTorres07) [inline README.md:1535]: Introduce other enter to make full stop ?

File: README.md [hunk 4]
```diff
@@ -1529,23 +1532,24 @@ $ helm del --purge tfm-almacar
...
+The application is able to store items and orders information, with the limitations of its data model described in previous chapters. This information is stored on a MySQL database but other database could be used just adapting one service, thanks to the usage of Spring Data JPA.
```
Comment (carlosTorres07) [inline README.md:1537]: Same comment for full stop ?

**Review by carlosTorres07 — APPROVED** [review]

**Pull request merged**